gravitee-apim-console-webui/src/management/application/details/general/application-general.component.ts

```diff
@@ -135,6 +135,9 @@ export class ApplicationGeneralComponent implements OnInit {
   onSubmit() {
     const imagesValue = this.applicationForm.getRawValue().images;
 
+    const pictureUrl = imagesValue?.picture?.[0]?.dataUrl || this.initialApplication.picture;
```

Comment:
what about :
`const pictureUrl = imagesValue?.picture?.[0]?.dataUrl || this.initialApplication.picture || null`

and juste have : 
```
picture: pictureUrl,
```